app/lib/widgets/home_bikes_view/bike_view.dart

```diff
@@ -56,6 +56,11 @@ class BikeView extends StatelessWidget {
                 ),
               ],
             ),
+            Image.network(
+              bike.links!.thumbnail,
+              fit: BoxFit.cover,
+              height: 200,
+            ),
```

Comment:
Can we remove this change (at least for now)

Reply:
> 

Done :)